borgmatic/hooks/pushover.py

```diff
@@ -0,0 +1,80 @@
+import logging
+
+import requests
+
+logger = logging.getLogger(__name__)
+
+
+EMERGENCY_PRIORITY = 2
+
+
+def initialize_monitor(
+    ping_url, config, config_filename, monitoring_log_level, dry_run
+):  # pragma: no cover
+    '''
+    No initialization is necessary for this monitor.
+    '''
+    pass
+
+
+def ping_monitor(hook_config, config, config_filename, state, monitoring_log_level, dry_run):
+    '''
+    Post a message to the configured Pushover application.
+    If this is a dry run, then don't actually update anything.
+    '''
+
+    run_states = hook_config.get('states', ['fail'])
+
+    if state.name.lower() not in run_states:
+        return
```

Comment:
Super minor, but: Does this case have any test coverage? You can tell by running `tox` and looking at the coverage output near the bottom.. It'll tell you about any lines that have zero test coverage.

Reply:
I didn't know that tox will show that. Very cool. I will figure out which missing test cases I need to add.

In the meantime, here is the test case requested: https://github.com/borgmatic-collective/borgmatic/pull/86/commits/3ab4b45041a26cf09ff7b991cb3650bd50a6a43f

Reply:
Yeah it's the [`coverage`](https://coverage.readthedocs.io/) package that supports that. Pretty handy.

Reply:
I found the missing tests. See here: https://github.com/borgmatic-collective/borgmatic/pull/86/commits/ea47704d86b9c3ebe3e412144f1c0d4459e9b79b

```
---------- coverage: platform linux, python 3.12.3-final-0 -----------
Name    Stmts   Miss  Cover   Missing
-------------------------------------
TOTAL    3656      0   100%
```